src/entry_points/gentest.py

```diff
@@ -0,0 +1,303 @@
+    request = RequestManager(config.remote_nodes[0])
+
+    print("Perform tx request: eth_get_transaction_by_hash(" + f"{transaction_hash}" + ")")
+    tr = request.eth_get_transaction_by_hash(transaction_hash)
+
+    print("Perform debug_trace_call")
+    state = request.debug_trace_call(tr)
+
+    print("Generate py test >> " + output_file)
+    test = make_test_template(tr, state)
+    with open(output_file, "w") as file:
+        file.write(test)
+
+    print(test)
+
+
+def make_test_template(
+    tr: "RequestManager.RemoteTransaction", state: Dict[Address, Account]
+) -> str:
+    """
+    Prepare the .py file template
+    """
+    test = PYTEST_TEMPLATE
+    test = test.replace(
+        "$HEADLINE_COMMENT",
+        "gentest autogenerated test with debug_traceCall of tx.hash " + tr.tr_hash,
+    )
+    test = test.replace("$TEST_NAME", "test_transaction_" + tr.tr_hash[2:])
+    test = test.replace(
+        "$TEST_COMMENT", "gentest autogenerated test for tx.hash " + tr.tr_hash[2:]
+    )
+
+    # Print a nice .py storage pre
+    pad = "            "
+    state_str = ""
+    for address, account in state.items():
+        if isinstance(account, dict):
+            account_obj = Account(**account)
+            state_str += '        "' + str(address) + '": Account(\n'
+            state_str += pad + "balance=" + str(account_obj.balance) + ",\n"
+            if address == tr.transaction.sender:
+                state_str += pad + "nonce=" + str(tr.transaction.nonce) + ",\n"
+            else:
+                state_str += pad + "nonce=" + str(account_obj.nonce) + ",\n"
+
+            if account_obj.code is None:
+                state_str += pad + 'code="0x",\n'
+            else:
+                state_str += pad + 'code="' + str(account_obj.code) + '",\n'
+            state_str += pad + "storage={\n"
+
+            if account_obj.storage is not None:
+                for record, value in account_obj.storage.items():
+                    state_str += pad + '    "' + str(record) + '" : "' + str(value) + '",\n'
+
+            state_str += pad + "}\n"
+            state_str += "        ),\n"
+    test = test.replace("$PRE", state_str)
+
+    # Print legacy transaction in .py
+    tr_str = ""
+    tr_str += pad + "ty=" + str(tr.transaction.ty) + ",\n"
+    tr_str += pad + "chain_id=" + str(tr.transaction.chain_id) + ",\n"
+    tr_str += pad + "nonce=" + str(tr.transaction.nonce) + ",\n"
+    tr_str += pad + 'to="' + str(tr.transaction.to) + '",\n'
+    tr_str += pad + "gas_price=" + str(tr.transaction.gas_price) + ",\n"
+    tr_str += pad + "protected=False,\n"
+    tr_str += pad + 'data="' + str(tr.transaction.data) + '",\n'
+    tr_str += pad + "gas_limit=" + str(tr.transaction.gas_limit) + ",\n"
+    tr_str += pad + "value=" + str(tr.transaction.value) + ",\n"
+    tr_str += pad + "v=" + str(tr.transaction.v) + ",\n"
+    tr_str += pad + "r=" + str(tr.transaction.r) + ",\n"
+    tr_str += pad + "s=" + str(tr.transaction.s) + ",\n"
```

Comment:
```suggestion
    for field, value in asdict(tr.transaction):
        if value is not None:
            tr_str += f"{pad}{field}={value},\n"
```

Reply:
No, here I can't print all fields from the dict